project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ object Dependencies {
   // WARNING: Please Scala update versions in PluginCross.scala too
   val scala212 = "2.12.20"
   val scala213 = "2.13.15"
-  val scala3 = "3.3.4"
+  val scala3 = "3.5.2"
```

Comment:
Does that mean that all `utils` are in Scala 3.5.2 and correspondingly Zinc has to bumped to Scala 3.5.2?

Reply:
Commented on the relevant discussion 🧵 https://github.com/sbt/sbt/discussions/7194#discussioncomment-11062161